<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PartLinkTypeEditDialog</class>
 <widget class="QDialog" name="PartLinkTypeEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1149</width>
    <height>694</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Part Link Types</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="mainSplitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QWidget" name="widget_20" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QListWidget" name="ltypeList">
             <property name="sortingEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_21" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QToolButton" name="ltypeAddButton">
                <property name="icon">
                 <iconset theme="list-add">
                  <normaloff>.</normaloff>.</iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="ltypeRemoveButton">
                <property name="icon">
                 <iconset theme="list-remove">
                  <normaloff>.</normaloff>.</iconset>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_9" native="true">
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="globalErrorLabel">
                <property name="palette">
                 <palette>
                  <active>
                   <colorrole role="WindowText">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>0</red>
                      <green>170</green>
                      <blue>0</blue>
                     </color>
                    </brush>
                   </colorrole>
                  </active>
                  <inactive>
                   <colorrole role="WindowText">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>0</red>
                      <green>170</green>
                      <blue>0</blue>
                     </color>
                    </brush>
                   </colorrole>
                  </inactive>
                  <disabled>
                   <colorrole role="WindowText">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>120</red>
                      <green>120</green>
                      <blue>120</blue>
                     </color>
                    </brush>
                   </colorrole>
                  </disabled>
                 </palette>
                </property>
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>No errors.</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QGroupBox" name="ltypeEditBox">
          <property name="title">
           <string>Edit Part Link Type</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
           <item>
            <widget class="QWidget" name="widget_2" native="true">
             <layout class="QFormLayout" name="formLayout">
              <property name="horizontalSpacing">
               <number>15</number>
              </property>
              <item row="0" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>ID</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QWidget" name="widget_19" native="true">
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QLineEdit" name="idField"/>
                 </item>
                 <item>
                  <widget class="QLabel" name="idOrigLabel">
                   <property name="text">
                    <string notr="true">Original: XXX</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_3" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1">
              <item>
               <widget class="QGroupBox" name="groupBox_3">
                <property name="title">
                 <string>Side A</string>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1">
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QWidget" name="widget_4" native="true">
                   <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,1">
                    <property name="leftMargin">
                     <number>0</number>
                    </property>
                    <property name="topMargin">
                     <number>0</number>
                    </property>
                    <property name="rightMargin">
                     <number>0</number>
                    </property>
                    <property name="bottomMargin">
                     <number>0</number>
                    </property>
                    <item>
                     <widget class="QWidget" name="widget_6" native="true">
                      <layout class="QFormLayout" name="formLayout_2">
                       <property name="horizontalSpacing">
                        <number>15</number>
                       </property>
                       <item row="0" column="0">
                        <widget class="QLabel" name="label_4">
                         <property name="text">
                          <string>Name</string>
                         </property>
                        </widget>
                       </item>
                       <item row="0" column="1">
                        <widget class="QLineEdit" name="aNameEdit">
                         <property name="maxLength">
                          <number>128</number>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </widget>
                    </item>
                    <item>
                     <widget class="QWidget" name="widget_7" native="true">
                      <layout class="QVBoxLayout" name="verticalLayout_5">
                       <item>
                        <widget class="QCheckBox" name="aShowBox">
                         <property name="text">
                          <string>Show</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QCheckBox" name="aEditableBox">
                         <property name="text">
                          <string>Editable</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QCheckBox" name="aDisplaySelListBox">
                         <property name="text">
                          <string>Display Selection List</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <spacer name="verticalSpacer">
                         <property name="orientation">
                          <enum>Qt::Vertical</enum>
                         </property>
                         <property name="sizeHint" stdset="0">
                          <size>
                           <width>0</width>
                           <height>0</height>
                          </size>
                         </property>
                        </spacer>
                       </item>
                      </layout>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <widget class="QWidget" name="widget_5" native="true">
                   <layout class="QVBoxLayout" name="verticalLayout_6">
                    <item>
                     <widget class="QWidget" name="widget_8" native="true">
                      <layout class="QVBoxLayout" name="verticalLayout_7">
                       <property name="leftMargin">
                        <number>0</number>
                       </property>
                       <property name="topMargin">
                        <number>0</number>
                       </property>
                       <property name="rightMargin">
                        <number>0</number>
                       </property>
                       <property name="bottomMargin">
                        <number>0</number>
                       </property>
                       <item>
                        <widget class="QCheckBox" name="aInvertBox">
                         <property name="text">
                          <string>Invert Selection</string>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </widget>
                    </item>
                    <item>
                     <widget class="QListWidget" name="aPcatList">
                      <property name="sortingEnabled">
                       <bool>true</bool>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_4">
                <property name="title">
                 <string>Side B</string>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,1">
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QWidget" name="widget_14" native="true">
                   <layout class="QVBoxLayout" name="verticalLayout_12" stretch="0,1">
                    <property name="leftMargin">
                     <number>0</number>
                    </property>
                    <property name="topMargin">
                     <number>0</number>
                    </property>
                    <property name="rightMargin">
                     <number>0</number>
                    </property>
                    <property name="bottomMargin">
                     <number>0</number>
                    </property>
                    <item>
                     <widget class="QWidget" name="widget_15" native="true">
                      <layout class="QFormLayout" name="formLayout_4">
                       <property name="horizontalSpacing">
                        <number>15</number>
                       </property>
                       <item row="0" column="0">
                        <widget class="QLabel" name="label_6">
                         <property name="text">
                          <string>Name</string>
                         </property>
                        </widget>
                       </item>
                       <item row="0" column="1">
                        <widget class="QLineEdit" name="bNameEdit">
                         <property name="maxLength">
                          <number>128</number>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </widget>
                    </item>
                    <item>
                     <widget class="QWidget" name="widget_16" native="true">
                      <layout class="QVBoxLayout" name="verticalLayout_13">
                       <item>
                        <widget class="QCheckBox" name="bShowBox">
                         <property name="text">
                          <string>Show</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QCheckBox" name="bEditableBox">
                         <property name="text">
                          <string>Editable</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QCheckBox" name="bDisplaySelListBox">
                         <property name="text">
                          <string>Display Selection List</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <spacer name="verticalSpacer_3">
                         <property name="orientation">
                          <enum>Qt::Vertical</enum>
                         </property>
                         <property name="sizeHint" stdset="0">
                          <size>
                           <width>0</width>
                           <height>0</height>
                          </size>
                         </property>
                        </spacer>
                       </item>
                      </layout>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <widget class="QWidget" name="widget_17" native="true">
                   <layout class="QVBoxLayout" name="verticalLayout_14">
                    <item>
                     <widget class="QWidget" name="widget_18" native="true">
                      <layout class="QVBoxLayout" name="verticalLayout_15">
                       <property name="leftMargin">
                        <number>0</number>
                       </property>
                       <property name="topMargin">
                        <number>0</number>
                       </property>
                       <property name="rightMargin">
                        <number>0</number>
                       </property>
                       <property name="bottomMargin">
                        <number>0</number>
                       </property>
                       <item>
                        <widget class="QCheckBox" name="bInvertBox">
                         <property name="text">
                          <string>Invert Selection</string>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </widget>
                    </item>
                    <item>
                     <widget class="QListWidget" name="bPcatList">
                      <property name="sortingEnabled">
                       <bool>true</bool>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Changes</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="electronicsdb::ChangelogWidget" name="clogWidget" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>electronicsdb::ChangelogWidget</class>
   <extends>QWidget</extends>
   <header location="global">electronicsdb/gui/util/ChangelogWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
